Helper/Api.php

```diff
@@ -149,7 +158,8 @@ public function logger($message)
 
     public function getSettingsVal($val)
     {
-        return $this->_scopeConfig->getValue($val, ScopeInterface::SCOPE_STORE);
+        $scope = ($this->request->getParam('store')) ?: $this->request->getParam('website');
+        return $this->_scopeConfig->getValue($val, ScopeInterface::SCOPE_STORE, $scope);
```

Comment:
Shouldn't the constant `ScopeInterface::SCOPE_WEBSITE` be used for website instead of `ScopeInterface::SCOPE_STORE`?